check_deps.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+has_command() {
+ which "$1" $1 > /dev/null 2>&1
+}
+
+
+build_liboqs() {
+	git clone -b main https://github.com/open-quantum-safe/liboqs.git
+	cd liboqs
+	mkdir -p build && cd build
+	cmake -GNinja .. -DBUILD_SHARED_LIBS=ON
+	ninja
+	sudo ninja install
+	cd ..
+	rm -rf liboqs
```

Comment:
I guess we don't need to remove liboqs folder at present , Pqcrypt needs structural changes and currently I am planning to use cloned liboqs folder to derive unique version of pqcrypt for issue #11